pyradiosky/skymodel.py

```diff
@@ -774,6 +770,17 @@ def set_spectral_type_params(self, spectral_type):
 
         self._set_spectral_type_params(spectral_type)
 
+    @property
+    def ncomponent_length_params(self):
+        """Iterate over ncomponent length paramters."""
+        param_list = []
```

Comment:
I think this could be a nice one-liner:

`param_list = (param for param  in self if getattr(self, param).form == ("Ncomponents", ))`